jobfunnel/glassdoor.py

```diff
@@ -33,6 +33,18 @@ def __init__(self, args):
             'Cache-Control': 'no-cache',
             'Connection': 'keep-alive'
         }
+        self.location_headers = {
+            'accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,'
+            'image/webp,*/*;q=0.01',
+            'accept-encoding': 'gzip, deflate, sdch, br',
+            'accept-language': 'en-GB,en-US;q=0.8,en;q=0.6',
```

Comment:
just curious but do they have an 'en-CAN' for canada? 

This also might have an interaction with the vectorizer vocab.

Reply:
It looks like they do. Will be added to `accept-language` when I push later today.